    except Exception as e:
        # Erro genérico
        print("❌ Erro inesperado:", str(e))
        traceback.print_exc()
        return {
            "statusCode": 500,
            "body": json.dumps({"erro": "Erro inesperado", "detalhes": str(e)})
        }
